### ISSUE TITLE
Matrix multiplication in softmax documentation carried out incorrectly

### ISSUE THREAD

**Issue by bradfordlynch** [tensorflow/tensorflow#114]
In the softmax documentation on page:

http://tensorflow.org/tutorials/mnist/beginners/index.md

There is a set of images showing an example of what the function is doing:

![image](https://cloud.githubusercontent.com/assets/5950006/11081692/e3c5dc3c-87ec-11e5-8e9a-6ed0b0ace7c6.png)

Then the "xs" are pulled out so that we have the vectors Wx. But the coefficients in the image above is incorrect. If the image below, also take form the docs, shows the true form then the equations in the above image should be:

y1 = W1,1**x1** + W1,2**x2** + W1,3**x3** + b1

instead of:

y1 = W1,1**x1** + W1,2**x1** + W1,3**x1** + b1

![image](https://cloud.githubusercontent.com/assets/5950006/11081715/38e3fdb6-87ed-11e5-8621-ac80926f2c03.png)

**Comment by vrv** [comment]
Thanks for the report!  We have a pending fix this, will be updated soon.  De-duping with https://github.com/tensorflow/tensorflow/issues/78

**Comment by bradfordlynch** [comment]
No problem, sorry for reporting a duplicate! I didn't see that one when I searched for the issue.
